odoo/addons/base/models/ir_actions_report.py

```diff
@@ -810,23 +810,23 @@ def _get_rendering_context_model(self):
 
     @api.model
     def _get_rendering_context(self, docids, data):
+        # access the report details with sudo() but evaluation context as current user
+        self_sudo = self.sudo()
+
         # If the report is using a custom model to render its html, we must use it.
         # Otherwise, fallback on the generic html rendering.
-        report_model = self._get_rendering_context_model()
+        report_model = self_sudo._get_rendering_context_model()
```

Comment:
Does passing sudo() has a future for version 13? Herewith is the attached link to the migration ticket. There is also one video attached to it demonstrating the refusal for accessing cost lines when the default website belongs to a different company than that of the production order's company.

https://www.odoo.com/web#id=2335874&action=333&active_id=70&model=project.task&view_type=form&cids=2&menu_id=4720

Reply:
@qsb-odoo this PR is for 14.0, your ticket is for 13.0. We do not plan to backport this as it is part of a bigger refactoring of access rights that was done between 13 and 14.0